style.css

```diff
@@ -132,6 +140,38 @@ body {
   color: var(--secondary-color);
 }
 
+.mobile-menu {
+  inset: 0 0 0 0%;
+  position: fixed;
+  padding: 5rem;
+  background: rgba(96, 112, 255, 0.7);
+  backdrop-filter: blur(0.7rem);
+  z-index: 100;
+  transform: translateX(100%);
+  transition: transform 0.4s;
+}
+
+.show-menu {
+  transform: translateX(0);
+}
+
+.mobile-menu ul {
+  display: flex;
+  flex-direction: column;
+}
+
+.nav-item-m {
+  margin-bottom: 2rem;
+  font-weight: 600;
+  font-size: 26px;
+  color: var(--white-color);
+  transition: color 0.3s ease-out;
+}
+
+.nav-item-m:hover {
+  color: var(--primary-color);
+}
+
 /* hero section */
 .hero {
   position: relative;
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** I think it would be better if you remove the **scrolling** on your **hero** section on some Desktop scene width to make your Design still stick s to the  Figma template. You can fix this issue by only adding the following property in the `.hero {...}` class in Desktop styling at **line 461**
```
.hero {
    ....
    overflow: hidden;
  }

```